src/containers/NavBar.tsx

```diff
@@ -74,11 +113,33 @@ export default function(props: NavBarConfig): JSX.Element {
         />
       )}
 
-      <Root position={props.position} theme={theme}>
-        <Flex>
-          <div style={{ flexGrow: 1 }}>{titleChildren}</div>
+      <Root
+        position={props.position}
+        theme={theme}
+        className={`${navOpen ? "nav-open" : ""}`}
+      >
+        <Flex className="children-container">
+          <div className="title-children-container" style={{ flexGrow: 1 }}>
+            {titleChildren}
+          </div>
           {flexChildren}
+          <Button
+            type="borderless"
+            className="nav-button"
+            onClick={() => {
+              setNavState(!navOpen);
+            }}
+          >
+            <Menu size={25} />
+          </Button>
         </Flex>
+        {navOpen && (
+          <div className="title-children-container-mobile">
+            {titleChildren.filter((child, index) => {
```

Comment:
When `title-children-container` is created, nothing ever populates

Reply:
navOpen is a state variable of this component that I created. When the user click the button on line 124 it sets this value to true or false, showing or not the mobile sideNav.

One time that I had some dificulties solving this problem only with CSS i thought it would be a good idea to create a replication of the titlechildren (except the first child because that one is always shown in all resolutions).

Maybe a good development could be when the use has the sideNav open and clicks outside of it closes is (seting the navOpen to false)

Reply:
<html lang="en" data-rh="lang">

<head>
    <style id="react-tooltip">
        .__react_component_tooltip {
            border-radius: 3px;
            display: inline-block;
            font-size: 13px;
            left: -999em;
            opacity: 0;
            padding: 8px 21px;
            position: fixed;
            pointer-events: none;
            transition: opacity 0.3s ease-out;
            top: -999em;
            visibility: hidden;
            z-index: 999
        }
        
        .__react_component_tooltip.allow_hover,
        .__react_component_tooltip.allow_click {
            pointer-events: auto
        }
        
        .__react_component_tooltip:before,
        .__react_component_tooltip:after {
            content: "";
            width: 0;
            height: 0;
            position: absolute
        }
        
        .__react_component_tooltip.show {
            opacity: 0.9;
            margin-top: 0px;
            margin-left: 0px;
            visibility: visible
        }
        
        .__react_component_tooltip.type-dark {
            color: #fff;
            background-color: #222
        }
        
        .__react_component_tooltip.type-dark.place-top:after {
            border-top-color: #222;
            border-top-style: solid;
            border-top-width: 6px
        }
        
        .__react_component_tooltip.type-dark.place-bottom:after {
            border-bottom-color: #222;
            border-bottom-style: solid;
            border-bottom-width: 6px
        }
        
        .__react_component_tooltip.type-dark.place-left:after {
            border-left-color: #222;
            border-left-style: solid;
            border-left-width: 6px
        }
        
        .__react_component_tooltip.type-dark.place-right:after {
            border-right-color: #222;
            border-right-style: solid;
            border-right-width: 6px
        }
        
        .__react_component_tooltip.type-dark.border {
            border: 1px solid #fff
        }
        
        .__react_component_tooltip.type-dark.border.place-top:before {
            border-top: 8px solid #fff
        }
        
        .__react_component_tooltip.type-dark.border.place-bottom:before {
            border-bottom: 8px solid #fff
        }
        
        .__react_component_tooltip.type-dark.border.place-left:before {
            border-left: 8px solid #fff
        }
        
        .__react_component_tooltip.type-dark.border.place-right:before {
            border-right: 8px solid #fff
        }
        
        .__react_component_tooltip.type-success {
            color: #fff;
            background-color: #8DC572
        }
        
        .__react_component_tooltip.type-success.place-top:after {
            border-top-color: #8DC572;
            border-top-style: solid;
            border-top-width: 6px
        }
        
        .__react_component_tooltip.type-success.place-bottom:after {
            border-bottom-color: #8DC572;
            border-bottom-style: solid;
            border-bottom-width: 6px
        }
        
        .__react_component_tooltip.type-success.place-left:after {
            border-left-color: #8DC572;
            border-left-style: solid;
            border-left-width: 6px
        }
        
        .__react_component_tooltip.type-success.place-right:after {
            border-right-color: #8DC572;
            border-right-style: solid;
            border-right-width: 6px
        }
        
        .__react_component_tooltip.type-success.border {
            border: 1px solid #fff
        }
        
        .__react_component_tooltip.type-success.border.place-top:before {
            border-top: 8px solid #fff
        }
        
        .__react_component_tooltip.type-success.border.place-bottom:before {
            border-bottom: 8px solid #fff
        }
        
        .__react_component_tooltip.type-success.border.place-left:before {
            border-left: 8px solid #fff
        }
        
        .__react_component_tooltip.type-success.border.place-right:before {
            border-right: 8px solid #fff
        }
        
        .__react_component_tooltip.type-warning {
            color: #fff;
            background-color: #F0AD4E
        }
        
        .__react_component_tooltip.type-warning.place-top:after {
            border-top-color: #F0AD4E;
            border-top-style: solid;
            border-top-width: 6px
        }
        
        .__react_component_tooltip.type-warning.place-bottom:after {
            border-bottom-color: #F0AD4E;
            border-bottom-style: solid;
            border-bottom-width: 6px
        }
        
        .__react_component_tooltip.type-warning.place-left:after {
            border-left-color: #F0AD4E;
            border-left-style: solid;
            border-left-width: 6px
        }
        
        .__react_component_tooltip.type-warning.place-right:after {
            border-right-color: #F0AD4E;
            border-right-style: solid;
            border-right-width: 6px
        }
        
        .__react_component_tooltip.type-warning.border {
            border: 1px solid #fff
        }
        
        .__react_component_tooltip.type-warning.border.place-top:before {
            border-top: 8px solid #fff
        }
        
        .__react_component_tooltip.type-warning.border.place-bottom:before {
            border-bottom: 8px solid #fff
        }
        
        .__react_component_tooltip.type-warning.border.place-left:before {
            border-left: 8px solid #fff
        }
        
        .__react_component_tooltip.type-warning.border.place-right:before {
            border-right: 8px solid #fff
        }
        
        .__react_component_tooltip.type-error {
            color: #fff;
            background-color: #BE6464
        }
        
        .__react_component_tooltip.type-error.place-top:after {
            border-top-color: #BE6464;
            border-top-style: solid;
            border-top-width: 6px
        }
        
        .__react_component_tooltip.type-error.place-bottom:after {
            border-bottom-color: #BE6464;
            border-bottom-style: solid;
            border-bottom-width: 6px
        }
        
        .__react_component_tooltip.type-error.place-left:after {
            border-left-color: #BE6464;
            border-left-style: solid;
            border-left-width: 6px
        }
        
        .__react_component_tooltip.type-error.place-right:after {
            border-right-color: #BE6464;
            border-right-style: solid;
            border-right-width: 6px
        }
        
        .__react_component_tooltip.type-error.border {
            border: 1px solid #fff
        }
        
        .__react_component_tooltip.type-error.border.place-top:before {
            border-top: 8px solid #fff
        }
        
        .__react_component_tooltip.type-error.border.place-bottom:before {
            border-bottom: 8px solid #fff
        }
        
        .__react_component_tooltip.type-error.border.place-left:before {
            border-left: 8px solid #fff
        }
        
        .__react_component_tooltip.type-error.border.place-right:before {
            border-right: 8px solid #fff
        }
        
        .__react_component_tooltip.type-info {
            color: #fff;
            background-color: #337AB7
        }
        
        .__react_component_tooltip.type-info.place-top:after {
            border-top-color: #337AB7;
            border-top-style: solid;
            border-top-width: 6px
        }
        
        .__react_component_tooltip.type-info.place-bottom:after {
            border-bottom-color: #337AB7;
            border-bottom-style: solid;
            border-bottom-width: 6px
        }
        
        .__react_component_tooltip.type-info.place-left:after {
            border-left-color: #337AB7;
            border-left-style: solid;
            border-left-width: 6px
        }
        
        .__react_component_tooltip.type-info.place-right:after {
            border-right-color: #337AB7;
            border-right-style: solid;
            border-right-width: 6px
        }
        
        .__react_component_tooltip.type-info.border {
            border: 1px solid #fff
        }
        
        .__react_component_tooltip.type-info.border.place-top:before {
            border-top: 8px solid #fff
        }
        
        .__react_component_tooltip.type-info.border.place-bottom:before {
            border-bottom: 8px solid #fff
        }
        
        .__react_component_tooltip.type-info.border.place-left:before {
            border-left: 8px solid #fff
        }
        
        .__react_component_tooltip.type-info.border.place-right:before {
            border-right: 8px solid #fff
        }
        
        .__react_component_tooltip.type-light {
            color: #222;
            background-color: #fff
        }
        
        .__react_component_tooltip.type-light.place-top:after {
            border-top-color: #fff;
            border-top-style: solid;
            border-top-width: 6px
        }
        
        .__react_component_tooltip.type-light.place-bottom:after {
            border-bottom-color: #fff;
            border-bottom-style: solid;
            border-bottom-width: 6px
        }
        
        .__react_component_tooltip.type-light.place-left:after {
            border-left-color: #fff;
            border-left-style: solid;
            border-left-width: 6px
        }
        
        .__react_component_tooltip.type-light.place-right:after {
            border-right-color: #fff;
            border-right-style: solid;
            border-right-width: 6px
        }
        
        .__react_component_tooltip.type-light.border {
            border: 1px solid #222
        }
        
        .__react_component_tooltip.type-light.border.place-top:before {
            border-top: 8px solid #222
        }
        
        .__react_component_tooltip.type-light.border.place-bottom:before {
            border-bottom: 8px solid #222
        }
        
        .__react_component_tooltip.type-light.border.place-left:before {
            border-left: 8px solid #222
        }
        
        .__react_component_tooltip.type-light.border.place-right:before {
            border-right: 8px solid #222
        }
        
        .__react_component_tooltip.place-top {
            margin-top: -10px
        }
        
        .__react_component_tooltip.place-top:before {
            border-left: 10px solid transparent;
            border-right: 10px solid transparent;
            bottom: -8px;
            left: 50%;
            margin-left: -10px
        }
        
        .__react_component_tooltip.place-top:after {
            border-left: 8px solid transparent;
            border-right: 8px solid transparent;
            bottom: -6px;
            left: 50%;
            margin-left: -8px
        }
        
        .__react_component_tooltip.place-bottom {
            margin-top: 10px
        }
        
        .__react_component_tooltip.place-bottom:before {
            border-left: 10px solid transparent;
            border-right: 10px solid transparent;
            top: -8px;
            left: 50%;
            margin-left: -10px
        }
        
        .__react_component_tooltip.place-bottom:after {
            border-left: 8px solid transparent;
            border-right: 8px solid transparent;
            top: -6px;
            left: 50%;
            margin-left: -8px
        }
        
        .__react_component_tooltip.place-left {
            margin-left: -10px
        }
        
        .__react_component_tooltip.place-left:before {
            border-top: 6px solid transparent;
            border-bottom: 6px solid transparent;
            right: -8px;
            top: 50%;
            margin-top: -5px
        }
        
        .__react_component_tooltip.place-left:after {
            border-top: 5px solid transparent;
            border-bottom: 5px solid transparent;
            right: -6px;
            top: 50%;
            margin-top: -4px
        }
        
        .__react_component_tooltip.place-right {
            margin-left: 10px
        }
        
        .__react_component_tooltip.place-right:before {
            border-top: 6px solid transparent;
            border-bottom: 6px solid transparent;
            left: -8px;
            top: 50%;
            margin-top: -5px
        }
        
        .__react_component_tooltip.place-right:after {
            border-top: 5px solid transparent;
            border-bottom: 5px solid transparent;
            left: -6px;
            top: 50%;
            margin-top: -4px
        }
        
        .__react_component_tooltip .multi-line {
            display: block;
            padding: 2px 0px;
            text-align: center
        }
    </style>
    <meta charset="utf-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="note" content="environment=development">
    <script src="/socket.io/socket.io.js"></script>
    <style type="text/css">
        @import url(https://fonts.googleapis.com/css?family=Open+Sans&display=swap);
    </style>
    <link type="text/css" rel="stylesheet" href="blob:http://localhost:3000/30f615f9-6058-4ae4-9604-8baf06ee194c">
    <style type="text/css">
        .styles_link__18O4d {
            text-decoration: none;
            color: initial;
        }
        
        .styles_wrapper__2Ttj5 {
            overflow-x: auto;
            padding: 2px;
            width: 100%;
            max-width: 100%;
        }
        
        .styles_table__28zJ6 {
            font-family: monospace, sans-serif;
            padding: 0;
            table-layout: auto;
            box-shadow: 0 0 0 1px #CED4DE;
            background-color: transparent;
            border-spacing: 0;
            border-collapse: collapse;
            border-style: hidden;
            border-radius: 2px;
            font-size: 14px;
            color: #13161F;
            display: block;
            width: 100%;
            overflow-x: initial;
            overflow-y: hidden;
        }
        
        .styles_table__28zJ6 thead {
            color: #2D3747;
            background: #EEF1F5;
        }
        
        .styles_table__28zJ6 thead th {
            font-weight: 400;
            padding: 20px 20px;
        }
        
        .styles_table__28zJ6 thead th:nth-of-type(1) {
            width: 20%;
        }
        
        .styles_table__28zJ6 thead th:nth-of-type(2) {
            width: 10%;
        }
        
        .styles_table__28zJ6 thead th:nth-of-type(3) {
            width: 10%;
        }
        
        .styles_table__28zJ6 thead th:nth-of-type(4) {
            width: 10%;
        }
        
        .styles_table__28zJ6 thead th:nth-of-type(5) {
            width: 20%;
        }
        
        .styles_table__28zJ6 tbody td {
            padding: 12px 20px;
            line-height: 2;
            font-weight: 200;
        }
        
        .styles_table__28zJ6 tbody > tr {
            display: table-row;
            border-top: 1px solid #CED4DE;
        }
        
        .styles_table__28zJ6 a {
            color: #0B5FFF;
        }
    </style>
    <style type="text/css">
        @import url(https://fonts.googleapis.com/css?family=Inconsolata&display=swap);
    </style>
    <style type="text/css">
        @import url(https://fonts.googleapis.com/css?family=Source+Sans+Pro&display=swap);
    </style>
    <link type="text/css" rel="stylesheet" href="blob:http://localhost:3000/84059a72-5b20-42de-b8f3-326c63682509">
    <style data-emotion="css-global">
        body {
            margin: 0;
            padding: 0;
        }
    </style>
    <style data-emotion="css-global">
        .icon-link {
            display: none;
        }
    </style>
    <style data-emotion="css-global">
        .with-overlay {
            overflow: hidden;
        }
    </style>
    <style data-emotion="css">
        .css-1ouyflx {
            font-family: 'Source Sans Pro', sans-serif;
            font-size: 20px;
            font-weight: 400;
            line-height: 1.5;
            color: var(--theme-ui-colors-text, #2D3747);
            background-color: var(--theme-ui-colors-background, #FFFFFF);
        }
    </style>
    <style data-emotion="css">
        .css-q1af2y-Layout > div {
            -webkit-flex: 1 1 auto;
            -ms-flex: 1 1 auto;
            flex: 1 1 auto;
        }
    </style>
    <style data-emotion="css">
        .css-jvit1p-Layout {
            min-height: 100vh;
            display: -webkit-box;
            display: -webkit-flex;
            display: -ms-flexbox;
            display: flex;
            -webkit-flex-direction: column;
            -ms-flex-direction: column;
            flex-direction: column;
        }
    </style>
    <style data-emotion="css">
        .css-jvit1p-Layout > div {
            -webkit-flex: 1 1 auto;
            -ms-flex: 1 1 auto;
            flex: 1 1 auto;
        }
    </style>
    <style data-emotion="css">
        .css-nlvt4-Layout {
            box-sizing: border-box;
            min-width: 0;
            min-height: 100vh;
            display: -webkit-box;
            display: -webkit-flex;
            display: -ms-flexbox;
            display: flex;
            -webkit-flex-direction: column;
            -ms-flex-direction: column;
            flex-direction: column;
        }
    </style>
    <style data-emotion="css">
        .css-nlvt4-Layout > div {
            -webkit-flex: 1 1 auto;
            -ms-flex: 1 1 auto;
            flex: 1 1 auto;
        }
    </style>
    <style data-emotion="css">
        .css-1rbeiz2-Layout {
            display: -webkit-box;
            display: -webkit-flex;
            display: -ms-flexbox;
            display: flex;
            -webkit-flex-direction: column;
            -ms-flex-direction: column;
            flex-direction: column;
            min-height: 100vh;
        }
    </style>
    <style data-emotion="css">
        .css-ni205y-Layout {
            -webkit-flex: 1 1 auto;
            -ms-flex: 1 1 auto;
            flex: 1 1 auto;
            display: -webkit-box;
            display: -webkit-flex;
            display: -ms-flexbox;
            display: flex;
            -webkit-flex-direction: column;
            -ms-flex-direction: column;
            flex-direction: column;
            min-height: 100vh;
        }
    </style>
    <style data-emotion="css">
        .css-jzir3m-Layout {
            box-sizing: border-box;
            min-width: 0;
            -webkit-flex: 1 1 auto;
            -ms-flex: 1 1 auto;
            flex: 1 1 auto;
            display: -webkit-box;
            display: -webkit-flex;
            display: -ms-flexbox;
            display: flex;
            -webkit-flex-direction: column;
            -ms-flex-direction: column;
            flex-direction: column;
            min-height: 100vh;
        }
    </style>
    <style data-emotion="css">
        .css-vawx9c-Layout {
            padding-top: 0;
            padding-bottom: 0;
            -webkit-flex: 1;
            -ms-flex: 1;
            flex: 1;
            display: grid;
            grid-template-columns: 250px 1fr;
            min-height: 100vh;
        }
    </style>
    <style data-emotion="css">
        @media screen and (max-width:57.5em) {
            .css-vawx9c-Layout {
                display: block;
            }
        }
    </style>
    <style data-emotion="css">
        .css-1tn7q1x {
            z-index: 999;
            position: fixed;
            top: 81px;
            right: 0;
            bottom: 0;
            left: 0;
            background: rgba(0, 0, 0, 0.6);
            -webkit-transition: all .2s ease-out;
            transition: all .2s ease-out;
            visibility: hidden;
            opacity: 0;
        }
    </style>
    <style data-emotion="css">
        .css-h4snl3 {
            box-sizing: border-box;
            min-width: 0;
            z-index: 999;
            position: fixed;
            top: 81px;
            right: 0;
            bottom: 0;
            left: 0;
            background: rgba(0, 0, 0, 0.6);
            -webkit-transition: all .2s ease-out;
            transition: all .2s ease-out;
            visibility: hidden;
            opacity: 0;
        }
    </style>
    <style data-emotion="css">
        .css-1ethd5x {
            padding-top: 32px;
            padding-bottom: 32px;
            padding-left: 32px;
            padding-right: 32px;
            -webkit-flex-direction: column;
            -ms-flex-direction: column;
            flex-direction: column;
            position: -webkit-sticky;
            position: sticky;
            top: 0;
            z-index: 1;
            min-width: 0;
            max-height: 100vh;
            border-right: 1px solid var(--theme-ui-colors-border, #CED4DE);
            overflow: auto;
            -webkit-overflow-scrolling: touch;
            background-color: var(--theme-ui-colors-sidebar-bg, #FFFFFF);
        }
    </style>
    <style data-emotion="css">
        @media screen and (max-width:57.5em) {
            .css-1ethd5x {
                z-index: 9999;
                display: block;
                position: fixed;
                top: 81px;
                left: 0;
                bottom: 0;
                width: 256px;
                padding-left: 32px;
                padding-right: 32px;
                background-color: var(--theme-ui-colors-background, #FFFFFF);
                -webkit-transition: -webkit-transform .2s ease-out;
                -webkit-transition: transform .2s ease-out;
                transition: transform .2s ease-out;
                -webkit-transform: translateX(-100%);
                -ms-transform: translateX(-100%);
                transform: translateX(-100%);
            }
        }
    </style>
    <style data-emotion="css">
        .css-1li9yso {
            box-sizing: border-box;
            min-width: 0;
            padding-top: 32px;
            padding-bottom: 32px;
            padding-left: 32px;
            padding-right: 32px;
            -webkit-flex-direction: column;
            -ms-flex-direction: column;
            flex-direction: column;
            position: -webkit-sticky;
            position: sticky;
            top: 0;
            z-index: 1;
            min-width: 0;
            max-height: 100vh;
            border-right: 1px solid var(--theme-ui-colors-border, #CED4DE);
            overflow: auto;
            -webkit-overflow-scrolling: touch;
            background-color: var(--theme-ui-colors-sidebar-bg, #FFFFFF);
        }
    </style>
    <style data-emotion="css">
        @media screen and (max-width:57.5em) {
            .css-1li9yso {
                z-index: 9999;
                display: block;
                position: fixed;
                top: 81px;
                left: 0;
                bottom: 0;
                width: 256px;
                padding-left: 32px;
                padding-right: 32px;
                background-color: var(--theme-ui-colors-background, #FFFFFF);
                -webkit-transition: -webkit-transform .2s ease-out;
                -webkit-transition: transform .2s ease-out;
                transition: transform .2s ease-out;
                -webkit-transform: translateX(-100%);
                -ms-transform: translateX(-100%);
                transform: translateX(-100%);
            }
        }
    </style>
    <style data-emotion="css">
        .css-1vb0vii-NavSearch {
            margin-bottom: 16px;
            display: -webkit-inline-box;
            display: -webkit-inline-flex;
            display: -ms-inline-flexbox;
            display: inline-flex;
            -webkit-align-items: center;
            -webkit-box-align: center;
            -ms-flex-align: center;
            align-items: center;
        }
    </style>
    <style data-emotion="css">
        .css-1cz2jww-NavSearch {
            color: var(--theme-ui-colors-border, #CED4DE);
            margin-right: 8px;
        }
    </style>
    <style data-emotion="css">
        .css-1f1wpfl-NavSearch {
            outline: none;
            background: none;
            border: none;
            color: var(--theme-ui-colors-text, #2D3747);
            font-size: 14px;
        }
    </style>
    <style data-emotion="css">
        .css-1rqvyud {
            margin-top: 8px;
            margin-bottom: 8px;
            display: block;
            color: var(--theme-ui-colors-sidebar-navGroup, #2D3747);
            -webkit-text-decoration: none;
            text-decoration: none;
            font-size: 16px;
        }
    </style>
    <style data-emotion="css">
        .css-1rqvyud.active {
            color: var(--theme-ui-colors-sidebar-navLinkActive, #0B5FFF);
        }
    </style>
    <style data-emotion="css">
        .css-1ooej63 {
            margin-top: 8px;
            margin-bottom: 8px;
            display: block;
            color: var(--theme-ui-colors-sidebar-tocLink, #67788a);
            -webkit-text-decoration: none;
            text-decoration: none;
            font-size: 14px;
            margin-left: 16px;
            position: relative;
        }
    </style>
    <style data-emotion="css">
        .css-1ooej63.active {
            color: var(--theme-ui-colors-sidebar-tocLinkActive, #1D2330);
        }
    </style>
    <style data-emotion="css">
        .css-1ooej63.active::before {
            content: "";
            position: absolute;
            display: block;
            top: 2px;
            left: -8px;
            height: 1rem;
            background-color: var(--theme-ui-colors-primary, #0B5FFF);
            -webkit-transition: width 200ms ease 0s;
            transition: width 200ms ease 0s;
            width: 2px;
            border-radius: 1px;
        }
    </style>
    <style data-emotion="css">
        .css-mm0xfh-NavGroup {
            margin-top: 16px;
            margin-bottom: 16px;
        }
    </style>
    <style data-emotion="css">
        .css-1jjvjl3-NavGroup {
            margin-bottom: 4px;
            font-size: 16px;
            font-weight: 500;
            color: var(--theme-ui-colors-sidebar-navGroup, #2D3747);
            cursor: pointer;
            display: -webkit-box;
            display: -webkit-flex;
            display: -ms-flexbox;
            display: flex;
            -webkit-box-pack: justify;
            -webkit-justify-content: space-between;
            -ms-flex-pack: justify;
            justify-content: space-between;
            -webkit-align-items: center;
            -webkit-box-align: center;
            -ms-flex-align: center;
            align-items: center;
        }
    </style>
    <style data-emotion="css">
        .css-nf86ym-NavGroup {
            margin-left: 4px;
            -webkit-flex-shrink: 0;
            -ms-flex-negative: 0;
            flex-shrink: 0;
            -webkit-align-self: baseline;
            -ms-flex-item-align: baseline;
            align-self: baseline;
            -webkit-transform: rotateX(0deg);
            -ms-transform: rotateX(0deg);
            transform: rotateX(0deg);
            -webkit-transform-origin: center;
            -ms-transform-origin: center;
            transform-origin: center;
            -webkit-transition: -webkit-transform .3s ease-in-out;
            -webkit-transition: transform .3s ease-in-out;
            transition: transform .3s ease-in-out;
        }
    </style>
    <style data-emotion="css">
        .css-eq2713-NavGroup {
            margin-left: 8px;
        }
    </style>
    <style data-emotion="css">
        .css-tmi5gp-Layout {
            background-color: var(--theme-ui-colors-background, #FFFFFF);
            position: relative;
            max-width: 1280px;
            padding-top: 48px;
            padding-bottom: 48px;
            padding-left: 32px;
            padding-right: 32px;
            padding: 32px;
        }
    </style>
    <style data-emotion="css">
        @media screen and (max-width:57.5em) {
            .css-tmi5gp-Layout {
                padding-top: 48px;
                padding-bottom: 32px;
                padding-left: 32px;
                padding-right: 32px;
            }
        }
    </style>
    <style data-emotion="css">
        .css-hwr2h6-Layout {
            width: 100%;
            min-width: 0;
            max-width: 1280px;
            margin-left: auto;
            margin-right: auto;
            padding: 32px;
            background-color: var(--theme-ui-colors-background, #FFFFFF);
            position: relative;
            max-width: 1280px;
            padding-top: 48px;
            padding-bottom: 48px;
            padding-left: 32px;
            padding-right: 32px;
            padding: 32px;
        }
    </style>
    <style data-emotion="css">
        @media screen and (max-width:57.5em) {
            .css-hwr2h6-Layout {
                padding-top: 48px;
                padding-bottom: 32px;
                padding-left: 32px;
                padding-right: 32px;
            }
        }
    </style>
    <style data-emotion="css">
        .css-1pglt8i-Layout {
            box-sizing: border-box;
            min-width: 0;
            width: 100%;
            min-width: 0;
            max-width: 1280px;
            margin-left: auto;
            margin-right: auto;
            padding: 32px;
            background-color: var(--theme-ui-colors-background, #FFFFFF);
            position: relative;
            max-width: 1280px;
            padding-top: 48px;
            padding-bottom: 48px;
            padding-left: 32px;
            padding-right: 32px;
            padding: 32px;
        }
    </style>
    <style data-emotion="css">
        @media screen and (max-width:57.5em) {
            .css-1pglt8i-Layout {
                padding-top: 48px;
                padding-bottom: 32px;
                padding-left: 32px;
                padding-right: 32px;
            }
        }
    </style>
    <style data-emotion="css">
        .css-1vci3bl {
            padding: 0;
            margin: 0;
            margin-bottom: 16px;
            font-size: 48px;
            font-family: 'Source Sans Pro', sans-serif;
            line-height: 1.125;
            font-weight: 700;
        }
    </style>
    <style data-emotion="css">
        .css-1a2e1jm {
            padding: 0;
            margin: 0;
            margin-bottom: 16px;
            max-width: 100%;
        }
    </style>
    <style data-emotion="css">
        .css-1ftn47p {
            padding: 0;
            margin: 0;
            margin-bottom: 16px;
        }
    </style>
    <style data-emotion="css">
        .css-1iv9jyz {
            color: var(--theme-ui-colors-primary, #0B5FFF);
            -webkit-text-decoration: none;
            text-decoration: none;
        }
    </style>
    <style data-emotion="css">
        .css-1iv9jyz:hover {
            color: secondary;
            -webkit-text-decoration: underline;
            text-decoration: underline;
        }
    </style>
    <style data-emotion="css">
        .css-1r0pe6c {
            color: inherit;
            -webkit-text-decoration: none;
            text-decoration: none;
        }
    </style>
    <style data-emotion="css">
        .css-1r0pe6c:hover {
            -webkit-text-decoration: underline;
            text-decoration: underline;
        }
    </style>
    <style data-emotion="css">
        .css-1h0sj8d {
            padding: 16px;
            margin: 0;
            margin-bottom: 32px;
            font-size: 85%;
            margin-top: 32px;
            text-align: left;
            font-family: Inconsolata;
            border-radius: 4px;
        }
    </style>
    <style data-emotion="css">
        .css-1h0sj8d dark {
            [object Object];
        }
    </style>
    <style data-emotion="css">
        .css-1h0sj8d light {
            [object Object];
        }
    </style>
    <style data-emotion="css">
        .css-1baulvz {
            display: inline-block;
        }
    </style>
    <style data-emotion="css">
        .css-1sc0ky7 {
            z-index: 999;
            position: fixed;
            top: 81px;
            right: 0;
            bottom: 0;
            left: 0;
            background: rgba(0, 0, 0, 0.6);
            -webkit-transition: all .2s ease-out;
            transition: all .2s ease-out;
            visibility: visible;
            opacity: 1;
        }
    </style>
    <style data-emotion="css">
        .css-1y7m1p3 {
            box-sizing: border-box;
            min-width: 0;
            z-index: 999;
            position: fixed;
            top: 81px;
            right: 0;
            bottom: 0;
            left: 0;
            background: rgba(0, 0, 0, 0.6);
            -webkit-transition: all .2s ease-out;
            transition: all .2s ease-out;
            visibility: visible;
            opacity: 1;
        }
    </style>
    <style data-emotion="css">
        .css-1hmd4uj {
            padding-top: 32px;
            padding-bottom: 32px;
            padding-left: 32px;
            padding-right: 32px;
            -webkit-flex-direction: column;
            -ms-flex-direction: column;
            flex-direction: column;
            position: -webkit-sticky;
            position: sticky;
            top: 0;
            z-index: 1;
            min-width: 0;
            max-height: 100vh;
            border-right: 1px solid var(--theme-ui-colors-border, #CED4DE);
            overflow: auto;
            -webkit-overflow-scrolling: touch;
            background-color: var(--theme-ui-colors-sidebar-bg, #FFFFFF);
        }
    </style>
    <style data-emotion="css">
        @media screen and (max-width:57.5em) {
            .css-1hmd4uj {
                z-index: 9999;
                display: block;
                position: fixed;
                top: 81px;
                left: 0;
                bottom: 0;
                width: 256px;
                padding-left: 32px;
                padding-right: 32px;
                background-color: var(--theme-ui-colors-background, #FFFFFF);
                -webkit-transition: -webkit-transform .2s ease-out;
                -webkit-transition: transform .2s ease-out;
                transition: transform .2s ease-out;
                -webkit-transform: translateX(0);
                -ms-transform: translateX(0);
                transform: translateX(0);
            }
        }
    </style>
    <style data-emotion="css">
        .css-a9pplo {
            box-sizing: border-box;
            min-width: 0;
            padding-top: 32px;
            padding-bottom: 32px;
            padding-left: 32px;
            padding-right: 32px;
            -webkit-flex-direction: column;
            -ms-flex-direction: column;
            flex-direction: column;
            position: -webkit-sticky;
            position: sticky;
            top: 0;
            z-index: 1;
            min-width: 0;
            max-height: 100vh;
            border-right: 1px solid var(--theme-ui-colors-border, #CED4DE);
            overflow: auto;
            -webkit-overflow-scrolling: touch;
            background-color: var(--theme-ui-colors-sidebar-bg, #FFFFFF);
        }
    </style>
    <style data-emotion="css">
        @media screen and (max-width:57.5em) {
            .css-a9pplo {
                z-index: 9999;
                display: block;
                position: fixed;
                top: 81px;
                left: 0;
                bottom: 0;
                width: 256px;
                padding-left: 32px;
                padding-right: 32px;
                background-color: var(--theme-ui-colors-background, #FFFFFF);
                -webkit-transition: -webkit-transform .2s ease-out;
                -webkit-transition: transform .2s ease-out;
                transition: transform .2s ease-out;
                -webkit-transform: translateX(0);
                -ms-transform: translateX(0);
                transform: translateX(0);
            }
        }
    </style>
    <style data-emotion="css">
        .css-1bns4z4-NavGroup {
            margin-left: 4px;
            -webkit-flex-shrink: 0;
            -ms-flex-negative: 0;
            flex-shrink: 0;
            -webkit-align-self: baseline;
            -ms-flex-item-align: baseline;
            align-self: baseline;
            -webkit-transform: rotateX(180deg);
            -ms-transform: rotateX(180deg);
            transform: rotateX(180deg);
            -webkit-transform-origin: center;
            -ms-transform-origin: center;
            transform-origin: center;
            -webkit-transition: -webkit-transform .3s ease-in-out;
            -webkit-transition: transform .3s ease-in-out;
            transition: transform .3s ease-in-out;
        }
    </style>
    <style data-emotion="css">
        .css-otk1yf-Playground {
            position: relative;
        }
    </style>
    <style data-emotion="css">
        .css-nxr1xe {
            height: auto;
            display: block;
            min-height: 100%;
            width: calc(100% - 2px);
            background-color: var(--theme-ui-colors-playground-bg, #FFFFFF);
            border: 1px solid var(--theme-ui-colors-playground-border, #CED4DE);
            border-radius: 4px 4px 0 0;
        }
    </style>
    <style data-emotion="css">
        .css-ql2n7q-Playground {
            z-index: 9;
            display: -webkit-box;
            display: -webkit-flex;
            display: -ms-flexbox;
            display: flex;
            position: absolute;
            bottom: -20px;
            right: 32px;
        }
    </style>
    <style data-emotion="css">
        .css-m0zonp-Playground {
            padding: 8px;
            outline: none;
            background: transparent;
            border: none;
            display: -webkit-box;
            display: -webkit-flex;
            display: -ms-flexbox;
            display: flex;
            -webkit-align-items: center;
            -webkit-box-align: center;
            -ms-flex-align: center;
            align-items: center;
            padding-top: 4px;
            padding-bottom: 4px;
            background-color: var(--theme-ui-colors-border, #CED4DE);
            color: var(--theme-ui-colors-muted, #67788a);
            border-radius: 0 0 3px 3px;
        }
    </style>
    <style data-emotion="css">
        .css-m0zonp-Playground:hover {
            cursor: pointer;
        }
    </style>
    <style data-emotion="css">
        .css-m0zonp-Playground ~ .css-m0zonp-Playground {
            margin-left: 4px;
        }
    </style>
    <style data-emotion="css">
        .css-nc33kp-Playground {
            margin: 0;
            padding-top: 8px;
            padding-bottom: 8px;
            padding-left: 16px;
            padding-right: 16px;
            background-color: #FF4757;
            font-size: 14px;
            color: var(--theme-ui-colors-white, #FFFFFF);
        }
    </style>
    <style data-styled="active" data-styled-version="5.0.1">
        .dfGigY {
            font-size: 1em;
            width: -webkit-fit-content;
            width: -moz-fit-content;
            width: fit-content;
            outline: 0;
            padding: 0;
            border: 0;
            background-color: transparent;
            height: calc(2em + 30px);
            font-family: "Open Sans", sans-serif;
            margin: 0.5em 1em;
            display: inline-block;
        }
        
        .dfGigY::-moz-focus-inner {
            border: 0;
        }
        
        .dfGigY:focus {
            outline: 0;
        }
        
        .iodBqI {
            overflow: hidden;
            width: -webkit-min-content;
            width: -moz-min-content;
            width: min-content;
            height: -webkit-min-content;
            height: -moz-min-content;
            height: min-content;
            margin: 0;
            border-radius: .25em;
            position: relative;
            padding: 0;
        }
        
        .gHqyKq {
            margin: 0;
            cursor: pointer;
            outline: 0;
            width: -webkit-min-content;
            width: -moz-min-content;
            width: min-content;
            padding: 0.25em 0.5em;
            border-radius: .25em;
            color: #16a085;
            background-color: transparent;
        }
        
        .lnmdIW {
            width: 5px;
            height: 5px;
            border-radius: 1000px;
            -webkit-transform: scale(0);
            -ms-transform: scale(0);
            transform: scale(0);
            opacity: 1;
            position: absolute;
            z-index: 100;
            cursor: pointer;
            background-color: #0000004c;
        }
        
        .cIySBv {
            width: 5px;
            height: 5px;
            border-radius: 1000px;
            -webkit-transform: scale(0);
            -ms-transform: scale(0);
            transform: scale(0);
            opacity: 1;
            position: absolute;
            z-index: 100;
            cursor: pointer;
            -webkit-animation: keaFRJ 0.5s linear;
            animation: keaFRJ 0.5s linear;
            background-color: #0000004c;
        }
        
        .esHljB {
            position: fixed;
            top: 0;
            left: 0;
            margin: 0;
            padding: 0.5em 2em;
            z-index: 1000;
            width: calc(100% - 4em);
            height: 3em;
            box-shadow: 0px 4px 10px -8px #16a085;
            background-color: #fff;
        }
        
        .esHljB > button {
            padding: 0 1em;
        }
        
        .esHljB .nav-button {
            display: none !important;
            position: absolute;
            right: 2em;
            top: 2em;
            -webkit-transform: translate(0, -50%);
            -ms-transform: translate(0, -50%);
            transform: translate(0, -50%);
        }
        
        .esHljB .title-children-container-mobile {
            display: none;
        }
        
        @media (max-width:768px) {
            .esHljB.nav-open {
                height: auto;
            }
            .esHljB .nav-button {
                display: block !important;
            }
            .esHljB .title-children-container > *:not(:first-child) {
                display: none !important;
            }
            .esHljB .title-children-container-mobile {
                display: -webkit-box;
                display: -webkit-flex;
                display: -ms-flexbox;
                display: flex;
                -webkit-flex-direction: column;
                -ms-flex-direction: column;
                flex-direction: column;
            }
        }
        
        .dUlnmP {
            position: relative;
            top: 0;
            left: 0;
            margin: 0;
            padding: 0.5em 2em;
            height: 3em;
            box-shadow: 0px 4px 10px -8px #16a085;
            background-color: #fff;
        }
        
        .dUlnmP > button {
            padding: 0 1em;
        }
        
        .dUlnmP .nav-button {
            display: none !important;
            position: absolute;
            right: 2em;
            top: 2em;
            -webkit-transform: translate(0, -50%);
            -ms-transform: translate(0, -50%);
            transform: translate(0, -50%);
        }
        
        .dUlnmP .title-children-container-mobile {
            display: none;
        }
        
        @media (max-width:768px) {
            .dUlnmP.nav-open {
                height: auto;
                z-index: 1;
            }
            .dUlnmP .nav-button {
                display: block !important;
            }
            .dUlnmP .title-children-container > *:not(:first-child) {
                display: none !important;
            }
            .dUlnmP .title-children-container-mobile {
                display: -webkit-box;
                display: -webkit-flex;
                display: -ms-flexbox;
                display: flex;
                -webkit-flex-direction: column;
                -ms-flex-direction: column;
                flex-direction: column;
            }
        }
        
        .cakrmJ {
            display: -webkit-box;
            display: -webkit-flex;
            display: -ms-flexbox;
            display: flex;
            height: 3em;
            -webkit-box-pack: end;
            -webkit-justify-content: flex-end;
            -ms-flex-pack: end;
            justify-content: flex-end;
            -webkit-flex-direction: row;
            -ms-flex-direction: row;
            flex-direction: row;
            -webkit-align-items: center;
            -webkit-box-align: center;
            -ms-flex-align: center;
            align-items: center;
        }
        
        .cakrmJ div {
            display: -webkit-box;
            display: -webkit-flex;
            display: -ms-flexbox;
            display: flex;
            -webkit-align-items: center;
            -webkit-box-align: center;
            -ms-flex-align: center;
            align-items: center;
            -webkit-flex-direction: row;
            -ms-flex-direction: row;
            flex-direction: row;
        }
        
        .koHsPO {
            display: none;
        }
        
        @media (max-width:920px) {
            .koHsPO {
                display: block;
            }
        }
        
        @-webkit-keyframes keaFRJ {
            100% {
                -webkit-transform: scale(40);
                -ms-transform: scale(40);
                transform: scale(40);
                opacity: 0;
            }
        }
        
        @keyframes keaFRJ {
            100% {
                -webkit-transform: scale(40);
                -ms-transform: scale(40);
                transform: scale(40);
                opacity: 0;
            }
        }
    </style>
    <title>NavBar | React Uniui</title>
    <meta name="description" content="My awesome app using docz" data-rh="true">
    <meta property="og:description" content="My awesome app using docz" data-rh="true">
    <meta property="og:type" content="website" data-rh="true">
    <meta name="twitter:card" content="summary" data-rh="true">
    <meta name="twitter:description" content="My awesome app using docz" data-rh="true">
    <meta property="og:title" content="NavBar" data-rh="true">
    <meta name="twitter:title" content="NavBar" data-rh="true">
</head>

<body>
    <div id="___gatsby">
        <div tabindex="-1" id="gatsby-focus-wrapper" style="outline: none;">
            <div class="css-1ouyflx">
                <div>
                    <div data-testid="layout" class="css-nlvt4-Layout">
                        <main class="css-jzir3m-Layout">
                            <div>
                                <div class="Header__MenuDiv-mAFcl koHsPO">
                                    <button style="outline: none; background: transparent; border: none; opacity: 0.5; cursor: pointer; top: 2em; position: fixed; z-index: 10000;">
                                        <svg xmlns="http://www.w3.org/2000/svg" width="25" height="25" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
                                            <line x1="3" y1="12" x2="21" y2="12"></line>
                                            <line x1="3" y1="6" x2="21" y2="6"></line>
                                            <line x1="3" y1="18" x2="21" y2="18"></line>
                                        </svg>
                                    </button>
                                </div>
                                <div>
                                    <div style="height: 3em; padding: 0.5em 2em;"></div>
                                    <div class="NavBar__Root-ixoTDM esHljB">
                                        <div class="NavBar__Flex-fgIUMt cakrmJ children-container">
                                            <div class="title-children-container" style="flex-grow: 1;"><img src="data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+PCEtLSBHZW5lcmF0b3I6IEdyYXZpdC5pbyAtLT48c3ZnIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHN0eWxlPSJpc29sYXRpb246aXNvbGF0ZSIgdmlld0JveD0iMzc3Ljg5IDU3Ny43MTIgNzYwLjYxIDI2OC43ODgiIHdpZHRoPSI3NjAuNjFwdCIgaGVpZ2h0PSIyNjguNzg4cHQiPjxkZWZzPjxjbGlwUGF0aCBpZD0iX2NsaXBQYXRoX1RUOFlNOGo5ZlR6NnAwakloUWlsTDdlN0toaks1Zk45Ij48cmVjdCB4PSIzNzcuODkiIHk9IjU3Ny43MTIiIHdpZHRoPSI3NjAuNjEiIGhlaWdodD0iMjY4Ljc4OCIvPjwvY2xpcFBhdGg+PC9kZWZzPjxnIGNsaXAtcGF0aD0idXJsKCNfY2xpcFBhdGhfVFQ4WU04ajlmVHo2cDBqSWhRaWxMN2U3S2hqSzVmTjkpIj48cGF0aCBkPSIgTSA5NzUuMDQzIDYzMS4wNTMgTCA5NzUuMDQzIDYzMS4wNTMgUSA5NzkuMzAxIDYzMS4wNTMgOTgyLjA1NSA2MzMuNjgzIEwgOTgyLjA1NSA2MzMuNjgzIEwgOTgyLjA1NSA2MzMuNjgzIFEgOTg0LjgxIDYzNi4zMTIgOTg0LjgxIDY0MC41NjkgTCA5ODQuODEgNjQwLjU2OSBMIDk4NC44MSA3NTguMjcgTCA5ODQuODEgNzU4LjI3IFEgOTg0LjgxIDc2Mi41MjggOTgyLjA1NSA3NjUuMjgyIEwgOTgyLjA1NSA3NjUuMjgyIEwgOTgyLjA1NSA3NjUuMjgyIFEgOTc5LjMwMSA3NjguMDM3IDk3NS4wNDMgNzY4LjAzNyBMIDk3NS4wNDMgNzY4LjAzNyBMIDk3NS4wNDMgNzY4LjAzNyBRIDk3MC43ODYgNzY4LjAzNyA5NjguMTU3IDc2NS4yODIgTCA5NjguMTU3IDc2NS4yODIgTCA5NjguMTU3IDc2NS4yODIgUSA5NjUuNTI3IDc2Mi41MjggOTY1LjUyNyA3NTguMjcgTCA5NjUuNTI3IDc1OC4yNyBMIDk2NS41MjcgNzQ5Ljc1NiBMIDk2NS41MjcgNzQ5Ljc1NiBRIDk1Ny4yNjMgNzU5LjAyMiA5NDUuNDkzIDc2NC4yODEgTCA5NDUuNDkzIDc2NC4yODEgTCA5NDUuNDkzIDc2NC4yODEgUSA5MzMuNzIzIDc2OS41NCA5MTkuOTQ5IDc2OS41NCBMIDkxOS45NDkgNzY5LjU0IEwgOTE5Ljk0OSA3NjkuNTQgUSA5MDIuMTY5IDc2OS41NCA4ODguMDIgNzYyLjAyNyBMIDg4OC4wMiA3NjIuMDI3IEwgODg4LjAyIDc2Mi4wMjcgUSA4NzMuODcgNzU0LjUxNCA4NjUuODU3IDc0MC4xMTQgTCA4NjUuODU3IDc0MC4xMTQgTCA4NjUuODU3IDc0MC4xMTQgUSA4NTcuODQzIDcyNS43MTUgODU3Ljg0MyA3MDUuNjgxIEwgODU3Ljg0MyA3MDUuNjgxIEwgODU3Ljg0MyA2NDAuNTY5IEwgODU3Ljg0MyA2NDAuNTY5IFEgODU3Ljg0MyA2MzYuNTYzIDg2MC41OTggNjMzLjgwOCBMIDg2MC41OTggNjMzLjgwOCBMIDg2MC41OTggNjMzLjgwOCBRIDg2My4zNTMgNjMxLjA1MyA4NjcuMzU5IDYzMS4wNTMgTCA4NjcuMzU5IDYzMS4wNTMgTCA4NjcuMzU5IDYzMS4wNTMgUSA4NzEuNjE3IDYzMS4wNTMgODc0LjM3MSA2MzMuODA4IEwgODc0LjM3MSA2MzMuODA4IEwgODc0LjM3MSA2MzMuODA4IFEgODc3LjEyNiA2MzYuNTYzIDg3Ny4xMjYgNjQwLjU2OSBMIDg3Ny4xMjYgNjQwLjU2OSBMIDg3Ny4xMjYgNzA1LjY4MSBMIDg3Ny4xMjYgNzA1LjY4MSBRIDg3Ny4xMjYgNzI3Ljk2OSA4ODkuNTIyIDczOS42MTQgTCA4ODkuNTIyIDczOS42MTQgTCA4ODkuNTIyIDczOS42MTQgUSA5MDEuOTE4IDc1MS4yNTggOTIxLjk1MyA3NTEuMjU4IEwgOTIxLjk1MyA3NTEuMjU4IEwgOTIxLjk1MyA3NTEuMjU4IFEgOTM0LjIyMyA3NTEuMjU4IDk0NC4xMTUgNzQ2LjUgTCA5NDQuMTE1IDc0Ni41IEwgOTQ0LjExNSA3NDYuNSBRIDk1NC4wMDcgNzQxLjc0MiA5NTkuNzY3IDczMy4zNTMgTCA5NTkuNzY3IDczMy4zNTMgTCA5NTkuNzY3IDczMy4zNTMgUSA5NjUuNTI3IDcyNC45NjMgOTY1LjUyNyA3MTQuNjk2IEwgOTY1LjUyNyA3MTQuNjk2IEwgOTY1LjUyNyA2NDAuNTY5IEwgOTY1LjUyNyA2NDAuNTY5IFEgOTY1LjUyNyA2MzYuMzEyIDk2OC4xNTcgNjMzLjY4MyBMIDk2OC4xNTcgNjMzLjY4MyBMIDk2OC4xNTcgNjMzLjY4MyBRIDk3MC43ODYgNjMxLjA1MyA5NzUuMDQzIDYzMS4wNTMgTCA5NzUuMDQzIDYzMS4wNTMgWiAgTSAxMDMzLjg5NCA2NDAuODIgTCAxMDMzLjg5NCA2NDAuODIgUSAxMDMzLjg5NCA2MzYuNTYzIDEwMzYuNjQ4IDYzMy44MDggTCAxMDM2LjY0OCA2MzMuODA4IEwgMTAzNi42NDggNjMzLjgwOCBRIDEwMzkuNDAzIDYzMS4wNTMgMTA0My42NjEgNjMxLjA1MyBMIDEwNDMuNjYxIDYzMS4wNTMgTCAxMDQzLjY2MSA2MzEuMDUzIFEgMTA0Ny45MTggNjMxLjA1MyAxMDUwLjU0NyA2MzMuODA4IEwgMTA1MC41NDcgNjMzLjgwOCBMIDEwNTAuNTQ3IDYzMy44MDggUSAxMDUzLjE3NyA2MzYuNTYzIDEwNTMuMTc3IDY0MC44MiBMIDEwNTMuMTc3IDY0MC44MiBMIDEwNTMuMTc3IDc1OC4yNyBMIDEwNTMuMTc3IDc1OC4yNyBRIDEwNTMuMTc3IDc2Mi41MjggMTA1MC41NDcgNzY1LjI4MiBMIDEwNTAuNTQ3IDc2NS4yODIgTCAxMDUwLjU0NyA3NjUuMjgyIFEgMTA0Ny45MTggNzY4LjAzNyAxMDQzLjY2MSA3NjguMDM3IEwgMTA0My42NjEgNzY4LjAzNyBMIDEwNDMuNjYxIDc2OC4wMzcgUSAxMDM5LjQwMyA3NjguMDM3IDEwMzYuNjQ4IDc2NS4yODIgTCAxMDM2LjY0OCA3NjUuMjgyIEwgMTAzNi42NDggNzY1LjI4MiBRIDEwMzMuODk0IDc2Mi41MjggMTAzMy44OTQgNzU4LjI3IEwgMTAzMy44OTQgNzU4LjI3IEwgMTAzMy44OTQgNjQwLjgyIFogIE0gMTA0My42NjEgNTc3LjcxMiBMIDEwNDMuNjYxIDU3Ny43MTIgUSAxMDQ5LjE3IDU3Ny43MTIgMTA1My4xNzcgNTgxLjQ2OCBMIDEwNTMuMTc3IDU4MS40NjggTCAxMDUzLjE3NyA1ODEuNDY4IFEgMTA1Ny4xODMgNTg1LjIyNSAxMDU3LjE4MyA1OTEuMjM1IEwgMTA1Ny4xODMgNTkxLjIzNSBMIDEwNTcuMTgzIDU5MS4yMzUgUSAxMDU3LjE4MyA1OTYuOTk1IDEwNTMuMTc3IDYwMS4wMDIgTCAxMDUzLjE3NyA2MDEuMDAyIEwgMTA1My4xNzcgNjAxLjAwMiBRIDEwNDkuMTcgNjA1LjAwOSAxMDQzLjQxIDYwNS4wMDkgTCAxMDQzLjQxIDYwNS4wMDkgTCAxMDQzLjQxIDYwNS4wMDkgUSAxMDM3LjY1IDYwNS4wMDkgMTAzMy43NjggNjAxLjAwMiBMIDEwMzMuNzY4IDYwMS4wMDIgTCAxMDMzLjc2OCA2MDEuMDAyIFEgMTAyOS44ODcgNTk2Ljk5NSAxMDI5Ljg4NyA1OTEuMjM1IEwgMTAyOS44ODcgNTkxLjIzNSBMIDEwMjkuODg3IDU5MS4yMzUgUSAxMDI5Ljg4NyA1ODUuMjI1IDEwMzMuODk0IDU4MS40NjggTCAxMDMzLjg5NCA1ODEuNDY4IEwgMTAzMy44OTQgNTgxLjQ2OCBRIDEwMzcuOTAxIDU3Ny43MTIgMTA0My42NjEgNTc3LjcxMiBMIDEwNDMuNjYxIDU3Ny43MTIgWiAiIGZpbGw9InJnYigyMiwxNjAsMTMzKSIvPjxwYXRoIGQ9IiBNIDU2My41OTEgNjMxLjA1MyBMIDU2My41OTEgNjMxLjA1MyBRIDU2Ny44NDggNjMxLjA1MyA1NzAuNjAzIDYzMy42ODMgTCA1NzAuNjAzIDYzMy42ODMgTCA1NzAuNjAzIDYzMy42ODMgUSA1NzMuMzU3IDYzNi4zMTIgNTczLjM1NyA2NDAuNTY5IEwgNTczLjM1NyA2NDAuNTY5IEwgNTczLjM1NyA3NTguMjcgTCA1NzMuMzU3IDc1OC4yNyBRIDU3My4zNTcgNzYyLjUyOCA1NzAuNjAzIDc2NS4yODIgTCA1NzAuNjAzIDc2NS4yODIgTCA1NzAuNjAzIDc2NS4yODIgUSA1NjcuODQ4IDc2OC4wMzcgNTYzLjU5MSA3NjguMDM3IEwgNTYzLjU5MSA3NjguMDM3IEwgNTYzLjU5MSA3NjguMDM3IFEgNTU5LjMzMyA3NjguMDM3IDU1Ni43MDQgNzY1LjI4MiBMIDU1Ni43MDQgNzY1LjI4MiBMIDU1Ni43MDQgNzY1LjI4MiBRIDU1NC4wNzQgNzYyLjUyOCA1NTQuMDc0IDc1OC4yNyBMIDU1NC4wNzQgNzU4LjI3IEwgNTU0LjA3NCA3NDkuNzU2IEwgNTU0LjA3NCA3NDkuNzU2IFEgNTQ1LjgxIDc1OS4wMjIgNTM0LjA0IDc2NC4yODEgTCA1MzQuMDQgNzY0LjI4MSBMIDUzNC4wNCA3NjQuMjgxIFEgNTIyLjI3IDc2OS41NCA1MDguNDk3IDc2OS41NCBMIDUwOC40OTcgNzY5LjU0IEwgNTA4LjQ5NyA3NjkuNTQgUSA0OTAuNzE2IDc2OS41NCA0NzYuNTY3IDc2Mi4wMjcgTCA0NzYuNTY3IDc2Mi4wMjcgTCA0NzYuNTY3IDc2Mi4wMjcgUSA0NjIuNDE4IDc1NC41MTQgNDU0LjQwNCA3NDAuMTE0IEwgNDU0LjQwNCA3NDAuMTE0IEwgNDU0LjQwNCA3NDAuMTE0IFEgNDQ2LjM5IDcyNS43MTUgNDQ2LjM5IDcwNS42ODEgTCA0NDYuMzkgNzA1LjY4MSBMIDQ0Ni4zOSA2NDAuNTY5IEwgNDQ2LjM5IDY0MC41NjkgUSA0NDYuMzkgNjM2LjU2MyA0NDkuMTQ1IDYzMy44MDggTCA0NDkuMTQ1IDYzMy44MDggTCA0NDkuMTQ1IDYzMy44MDggUSA0NTEuOSA2MzEuMDUzIDQ1NS45MDcgNjMxLjA1MyBMIDQ1NS45MDcgNjMxLjA1MyBMIDQ1NS45MDcgNjMxLjA1MyBRIDQ2MC4xNjQgNjMxLjA1MyA0NjIuOTE5IDYzMy44MDggTCA0NjIuOTE5IDYzMy44MDggTCA0NjIuOTE5IDYzMy44MDggUSA0NjUuNjczIDYzNi41NjMgNDY1LjY3MyA2NDAuNTY5IEwgNDY1LjY3MyA2NDAuNTY5IEwgNDY1LjY3MyA3MDUuNjgxIEwgNDY1LjY3MyA3MDUuNjgxIFEgNDY1LjY3MyA3MjcuOTY5IDQ3OC4wNjkgNzM5LjYxNCBMIDQ3OC4wNjkgNzM5LjYxNCBMIDQ3OC4wNjkgNzM5LjYxNCBRIDQ5MC40NjYgNzUxLjI1OCA1MTAuNSA3NTEuMjU4IEwgNTEwLjUgNzUxLjI1OCBMIDUxMC41IDc1MS4yNTggUSA1MjIuNzcxIDc1MS4yNTggNTMyLjY2MyA3NDYuNSBMIDUzMi42NjMgNzQ2LjUgTCA1MzIuNjYzIDc0Ni41IFEgNTQyLjU1NSA3NDEuNzQyIDU0OC4zMTUgNzMzLjM1MyBMIDU0OC4zMTUgNzMzLjM1MyBMIDU0OC4zMTUgNzMzLjM1MyBRIDU1NC4wNzQgNzI0Ljk2MyA1NTQuMDc0IDcxNC42OTYgTCA1NTQuMDc0IDcxNC42OTYgTCA1NTQuMDc0IDY0MC41NjkgTCA1NTQuMDc0IDY0MC41NjkgUSA1NTQuMDc0IDYzNi4zMTIgNTU2LjcwNCA2MzMuNjgzIEwgNTU2LjcwNCA2MzMuNjgzIEwgNTU2LjcwNCA2MzMuNjgzIFEgNTU5LjMzMyA2MzEuMDUzIDU2My41OTEgNjMxLjA1MyBMIDU2My41OTEgNjMxLjA1MyBaICBNIDY4My4yOTUgNjI5LjU1MSBMIDY4My4yOTUgNjI5LjU1MSBRIDcwMS4wNzUgNjI5LjU1MSA3MTUuMjI1IDYzNy4wNjMgTCA3MTUuMjI1IDYzNy4wNjMgTCA3MTUuMjI1IDYzNy4wNjMgUSA3MjkuMzc0IDY0NC41NzYgNzM3LjM4NyA2NTguOTc2IEwgNzM3LjM4NyA2NTguOTc2IEwgNzM3LjM4NyA2NTguOTc2IFEgNzQ1LjQwMSA2NzMuMzc1IDc0NS40MDEgNjkzLjQxIEwgNzQ1LjQwMSA2OTMuNDEgTCA3NDUuNDAxIDc1OC41MjEgTCA3NDUuNDAxIDc1OC41MjEgUSA3NDUuNDAxIDc2Mi41MjggNzQyLjY0NiA3NjUuMjgyIEwgNzQyLjY0NiA3NjUuMjgyIEwgNzQyLjY0NiA3NjUuMjgyIFEgNzM5Ljg5MiA3NjguMDM3IDczNS44ODUgNzY4LjAzNyBMIDczNS44ODUgNzY4LjAzNyBMIDczNS44ODUgNzY4LjAzNyBRIDczMS42MjggNzY4LjAzNyA3MjguODczIDc2NS4yODIgTCA3MjguODczIDc2NS4yODIgTCA3MjguODczIDc2NS4yODIgUSA3MjYuMTE4IDc2Mi41MjggNzI2LjExOCA3NTguNTIxIEwgNzI2LjExOCA3NTguNTIxIEwgNzI2LjExOCA2OTMuNDEgTCA3MjYuMTE4IDY5My40MSBRIDcyNi4xMTggNjcxLjEyMiA3MTMuNzIyIDY1OS40NzcgTCA3MTMuNzIyIDY1OS40NzcgTCA3MTMuNzIyIDY1OS40NzcgUSA3MDEuMzI2IDY0Ny44MzIgNjgxLjI5MiA2NDcuODMyIEwgNjgxLjI5MiA2NDcuODMyIEwgNjgxLjI5MiA2NDcuODMyIFEgNjY5LjAyMSA2NDcuODMyIDY1OS4xMjkgNjUyLjU5IEwgNjU5LjEyOSA2NTIuNTkgTCA2NTkuMTI5IDY1Mi41OSBRIDY0OS4yMzcgNjU3LjM0OCA2NDMuNDc3IDY2NS43MzcgTCA2NDMuNDc3IDY2NS43MzcgTCA2NDMuNDc3IDY2NS43MzcgUSA2MzcuNzE3IDY3NC4xMjcgNjM3LjcxNyA2ODQuMzk0IEwgNjM3LjcxNyA2ODQuMzk0IEwgNjM3LjcxNyA3NTguNTIxIEwgNjM3LjcxNyA3NTguNTIxIFEgNjM3LjcxNyA3NjIuNzc4IDYzNS4wODggNzY1LjQwOCBMIDYzNS4wODggNzY1LjQwOCBMIDYzNS4wODggNzY1LjQwOCBRIDYzMi40NTggNzY4LjAzNyA2MjguMjAxIDc2OC4wMzcgTCA2MjguMjAxIDc2OC4wMzcgTCA2MjguMjAxIDc2OC4wMzcgUSA2MjMuOTQ0IDc2OC4wMzcgNjIxLjE4OSA3NjUuNDA4IEwgNjIxLjE4OSA3NjUuNDA4IEwgNjIxLjE4OSA3NjUuNDA4IFEgNjE4LjQzNCA3NjIuNzc4IDYxOC40MzQgNzU4LjUyMSBMIDYxOC40MzQgNzU4LjUyMSBMIDYxOC40MzQgNjQwLjgyIEwgNjE4LjQzNCA2NDAuODIgUSA2MTguNDM0IDYzNi41NjMgNjIxLjE4OSA2MzMuODA4IEwgNjIxLjE4OSA2MzMuODA4IEwgNjIxLjE4OSA2MzMuODA4IFEgNjIzLjk0NCA2MzEuMDUzIDYyOC4yMDEgNjMxLjA1MyBMIDYyOC4yMDEgNjMxLjA1MyBMIDYyOC4yMDEgNjMxLjA1MyBRIDYzMi40NTggNjMxLjA1MyA2MzUuMDg4IDYzMy44MDggTCA2MzUuMDg4IDYzMy44MDggTCA2MzUuMDg4IDYzMy44MDggUSA2MzcuNzE3IDYzNi41NjMgNjM3LjcxNyA2NDAuODIgTCA2MzcuNzE3IDY0MC44MiBMIDYzNy43MTcgNjQ5LjMzNCBMIDYzNy43MTcgNjQ5LjMzNCBRIDY0NS45ODEgNjQwLjA2OSA2NTcuNzUxIDYzNC44MSBMIDY1Ny43NTEgNjM0LjgxIEwgNjU3Ljc1MSA2MzQuODEgUSA2NjkuNTIxIDYyOS41NTEgNjgzLjI5NSA2MjkuNTUxIEwgNjgzLjI5NSA2MjkuNTUxIFogIE0gNzkxLjk4MSA2NDAuODIgTCA3OTEuOTgxIDY0MC44MiBRIDc5MS45ODEgNjM2LjU2MyA3OTQuNzM1IDYzMy44MDggTCA3OTQuNzM1IDYzMy44MDggTCA3OTQuNzM1IDYzMy44MDggUSA3OTcuNDkgNjMxLjA1MyA4MDEuNzQ3IDYzMS4wNTMgTCA4MDEuNzQ3IDYzMS4wNTMgTCA4MDEuNzQ3IDYzMS4wNTMgUSA4MDYuMDA1IDYzMS4wNTMgODA4LjYzNCA2MzMuODA4IEwgODA4LjYzNCA2MzMuODA4IEwgODA4LjYzNCA2MzMuODA4IFEgODExLjI2NCA2MzYuNTYzIDgxMS4yNjQgNjQwLjgyIEwgODExLjI2NCA2NDAuODIgTCA4MTEuMjY0IDc1OC4yNyBMIDgxMS4yNjQgNzU4LjI3IFEgODExLjI2NCA3NjIuNTI4IDgwOC42MzQgNzY1LjI4MiBMIDgwOC42MzQgNzY1LjI4MiBMIDgwOC42MzQgNzY1LjI4MiBRIDgwNi4wMDUgNzY4LjAzNyA4MDEuNzQ3IDc2OC4wMzcgTCA4MDEuNzQ3IDc2OC4wMzcgTCA4MDEuNzQ3IDc2OC4wMzcgUSA3OTcuNDkgNzY4LjAzNyA3OTQuNzM1IDc2NS4yODIgTCA3OTQuNzM1IDc2NS4yODIgTCA3OTQuNzM1IDc2NS4yODIgUSA3OTEuOTgxIDc2Mi41MjggNzkxLjk4MSA3NTguMjcgTCA3OTEuOTgxIDc1OC4yNyBMIDc5MS45ODEgNjQwLjgyIFogIE0gODAxLjc0NyA1NzcuNzEyIEwgODAxLjc0NyA1NzcuNzEyIFEgODA3LjI1NyA1NzcuNzEyIDgxMS4yNjQgNTgxLjQ2OCBMIDgxMS4yNjQgNTgxLjQ2OCBMIDgxMS4yNjQgNTgxLjQ2OCBRIDgxNS4yNyA1ODUuMjI1IDgxNS4yNyA1OTEuMjM1IEwgODE1LjI3IDU5MS4yMzUgTCA4MTUuMjcgNTkxLjIzNSBRIDgxNS4yNyA1OTYuOTk1IDgxMS4yNjQgNjAxLjAwMiBMIDgxMS4yNjQgNjAxLjAwMiBMIDgxMS4yNjQgNjAxLjAwMiBRIDgwNy4yNTcgNjA1LjAwOSA4MDEuNDk3IDYwNS4wMDkgTCA4MDEuNDk3IDYwNS4wMDkgTCA4MDEuNDk3IDYwNS4wMDkgUSA3OTUuNzM3IDYwNS4wMDkgNzkxLjg1NSA2MDEuMDAyIEwgNzkxLjg1NSA2MDEuMDAyIEwgNzkxLjg1NSA2MDEuMDAyIFEgNzg3Ljk3NCA1OTYuOTk1IDc4Ny45NzQgNTkxLjIzNSBMIDc4Ny45NzQgNTkxLjIzNSBMIDc4Ny45NzQgNTkxLjIzNSBRIDc4Ny45NzQgNTg1LjIyNSA3OTEuOTgxIDU4MS40NjggTCA3OTEuOTgxIDU4MS40NjggTCA3OTEuOTgxIDU4MS40NjggUSA3OTUuOTg4IDU3Ny43MTIgODAxLjc0NyA1NzcuNzEyIEwgODAxLjc0NyA1NzcuNzEyIFogIiBmaWxsPSJyZ2IoMjYsMTg4LDE1NikiLz48ZGVmcz48ZmlsdGVyIGlkPSJ1WGYyVkhyd1p1SGFseXU4ek1pc0lQcUdYaWJjVDcxRiIgeD0iLTIwMCUiIHk9Ii0yMDAlIiB3aWR0aD0iNDAwJSIgaGVpZ2h0PSI0MDAlIiBmaWx0ZXJVbml0cz0ib2JqZWN0Qm91bmRpbmdCb3giIGNvbG9yLWludGVycG9sYXRpb24tZmlsdGVycz0ic1JHQiI+PGZlR2F1c3NpYW5CbHVyIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgaW49IlNvdXJjZUdyYXBoaWMiIHN0ZERldmlhdGlvbj0iMTcuMTc0NDM2MjUxMzU2MTEzIi8+PGZlT2Zmc2V0IHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgZHg9IjAiIGR5PSIwIiByZXN1bHQ9InBmXzEwMF9vZmZzZXRCbHVyIi8+PGZlRmxvb2QgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiBmbG9vZC1jb2xvcj0iI0U5NUY3QSIgZmxvb2Qtb3BhY2l0eT0iMC44NiIvPjxmZUNvbXBvc2l0ZSB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGluMj0icGZfMTAwX29mZnNldEJsdXIiIG9wZXJhdG9yPSJpbiIgcmVzdWx0PSJwZl8xMDBfZHJvcFNoYWRvdyIvPjxmZUJsZW5kIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgaW49IlNvdXJjZUdyYXBoaWMiIGluMj0icGZfMTAwX2Ryb3BTaGFkb3ciIG1vZGU9Im5vcm1hbCIvPjwvZmlsdGVyPjwvZGVmcz48ZyBmaWx0ZXI9InVybCgjdVhmMlZIcndadUhhbHl1OHpNaXNJUHFHWGliY1Q3MUYpIj48cGF0aCBkPSIgTSAxMDcxLjUgNzMzLjM1IEwgMTA3MS41IDcyNS44NSBMIDEwODYuNSA3MjUuODUgTCAxMDg2LjUgNzMzLjM1IEwgMTA4Ni41IDc5NC41IEwgNDM3LjM5IDc5NC41IEwgNDI5Ljg5IDc5NC41IEwgNDI5Ljg5IDc3OS41IEwgNDM3LjM5IDc3OS41IEwgMTA3MS41IDc3OS41IEwgMTA3MS41IDczMy4zNSBaICIgZmlsbD0icmdiKDIzMyw5NSwxMjIpIi8+PC9nPjwvZz48L3N2Zz4=" alt="Uniui" style="height: 2em;">
                                                <a href="https://github.com/ericm/uniui">
                                                    <div style="display: inline-block;">
                                                        <button class="Button__Root-lpafWu dfGigY">
                                                            <div class="Button__Container-eLxmbr iodBqI">
                                                                <div class="Button__ButtonStyled-cWqOxB gHqyKq">GitHub</div>
                                                                <div class="Button__Ripple-geatGG lnmdIW" style="top: 0px; left: 0px;"></div>
                                                            </div>
                                                        </button>
                                                    </div>
                                                </a>
                                                <a href="https://www.npmjs.com/package/react-uniui">
                                                    <div style="display: inline-block;">
                                                        <button class="Button__Root-lpafWu dfGigY">
                                                            <div class="Button__Container-eLxmbr iodBqI">
                                                                <div class="Button__ButtonStyled-cWqOxB gHqyKq">NPM</div>
                                                                <div class="Button__Ripple-geatGG lnmdIW" style="top: 0px; left: 0px;"></div>
                                                            </div>
                                                        </button>
                                                    </div>
                                                </a>
                                            </div>
                                            <div class="nav-button" style="display: inline-block;">
                                                <button class="Button__Root-lpafWu dfGigY">
                                                    <div class="Button__Container-eLxmbr iodBqI">
                                                        <div class="Button__ButtonStyled-cWqOxB gHqyKq">
                                                            <svg xmlns="http://www.w3.org/2000/svg" width="25" height="25" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
                                                                <line x1="3" y1="12" x2="21" y2="12"></line>
                                                                <line x1="3" y1="6" x2="21" y2="6"></line>
                                                                <line x1="3" y1="18" x2="21" y2="18"></line>
                                                            </svg>
                                                        </div>
                                                        <div class="Button__Ripple-geatGG lnmdIW" style="top: 39px; left: 26px;"></div>
                                                    </div>
                                                </button>
                                            </div>
                                        </div>
                                    </div>
                                </div>
                            </div>
                            <div class="css-vawx9c-Layout">
                                <div class="css-h4snl3"></div>
                                <div data-testid="sidebar" class="css-1li9yso">
                                    <div data-testid="nav-search" class="css-1vb0vii-NavSearch">
                                        <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="css-1cz2jww-NavSearch">
                                            <circle cx="11" cy="11" r="8"></circle>
                                            <line x1="21" y1="21" x2="16.65" y2="16.65"></line>
                                        </svg>
                                        <input placeholder="Type to search..." class="css-1f1wpfl-NavSearch" value="">
                                    </div><a class="css-1rqvyud" href="/">1. Quick Start</a>
                                    <div data-testid="nav-group" class="css-mm0xfh-NavGroup">
                                        <div class="css-1jjvjl3-NavGroup">2. Components
                                            <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="css-nf86ym-NavGroup">
                                                <polyline points="6 9 12 15 18 9"></polyline>
                                            </svg>
                                        </div>
                                        <div data-testid="nav-group-links" class="css-eq2713-NavGroup"></div>
                                    </div><a class="css-1rqvyud" href="/theming">3. Theming</a>
                                    <div data-testid="nav-group" class="css-mm0xfh-NavGroup">
                                        <div class="css-1jjvjl3-NavGroup">4. Containers
                                            <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="css-1bns4z4-NavGroup">
                                                <polyline points="6 9 12 15 18 9"></polyline>
                                            </svg>
                                        </div>
                                        <div data-testid="nav-group-links" class="css-eq2713-NavGroup"><a class="css-1rqvyud" href="/containers/box">Box</a><a aria-current="page" class="css-1rqvyud active" href="/containers/navbar">NavBar</a><a class=" css-1ooej63" href="/containers/navbar#basic-usage">Basic Usage</a></div>
                                    </div><a class="css-1rqvyud" href="/readme">Readme</a></div>
                                <div data-testid="main-container" class="css-1pglt8i-Layout">
                                    <h1 id="navbar-component" class="css-1vci3bl">NavBar Component</h1>
                                    <div class="styles_wrapper__2Ttj5">
                                        <table class="styles_table__28zJ6">
                                            <thead style="text-align: left;">
                                                <tr>
                                                    <th>Property</th>
                                                    <th>Type</th>
                                                    <th>Required</th>
                                                    <th>Default</th>
                                                </tr>
                                            </thead>
                                            <tbody>
                                                <tr>
                                                    <td>position</td>
                                                    <td><a class="styles_link__18O4d" data-tip="true" data-for="&quot;\&quot;relative\&quot; | \&quot;fixed\&quot;&quot;" href="#" currentitem="false">Enum</a>
                                                        <div class="__react_component_tooltip place-top type-dark " id="&quot;\&quot;relative\&quot; | \&quot;fixed\&quot;&quot;" data-id="tooltip">"relative" | "fixed"</div>
                                                    </td>
                                                    <td>true</td>
                                                    <td><em>-</em></td>
                                                </tr>
                                            </tbody>
                                        </table>
                                    </div>
                                    <h2 id="basic-usage"><a href="#basic-usage" class="css-1r0pe6c">Basic Usage</a></h2>
                                    <div data-testid="playground" style="position: relative; user-select: auto; margin: 0px auto 0px 0px; width: 100%; height: auto; max-width: 100%; min-width: 260px; box-sizing: border-box; flex-shrink: 0;">
                                        <div class="css-otk1yf-Playground">
                                            <div class="css-nxr1xe">
                                                <div data-testid="live-preview">
                                                    <div style="height: 10em;">
                                                        <div>
                                                            <div class="NavBar__Root-ixoTDM dUlnmP nav-open">
                                                                <div class="NavBar__Flex-fgIUMt cakrmJ children-container">
                                                                    <div class="title-children-container" style="flex-grow: 1;">
                                                                        <h1>Test</h1>
                                                                        <div style="display: inline-block;">
                                                                            <button class="Button__Root-lpafWu dfGigY">
                                                                                <div class="Button__Container-eLxmbr iodBqI">
                                                                                    <div class="Button__ButtonStyled-cWqOxB gHqyKq">Home</div>
                                                                                    <div class="Button__Ripple-geatGG lnmdIW" style="top: 0px; left: 0px;"></div>
                                                                                </div>
                                                                            </button>
                                                                        </div>
                                                                        <div style="display: inline-block;">
                                                                            <button class="Button__Root-lpafWu dfGigY">
                                                                                <div class="Button__Container-eLxmbr iodBqI">
                                                                                    <div class="Button__ButtonStyled-cWqOxB gHqyKq">About</div>
                                                                                    <div class="Button__Ripple-geatGG lnmdIW" style="top: 0px; left: 0px;"></div>
                                                                                </div>
                                                                            </button>
                                                                        </div>
                                                                    </div>
                                                                    <div class="nav-button" style="display: inline-block;">
                                                                        <button class="Button__Root-lpafWu dfGigY">
                                                                            <div class="Button__Container-eLxmbr iodBqI">
                                                                                <div class="Button__ButtonStyled-cWqOxB gHqyKq">
                                                                                    <svg xmlns="http://www.w3.org/2000/svg" width="25" height="25" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
                                                                                        <line x1="3" y1="12" x2="21" y2="12"></line>
                                                                                        <line x1="3" y1="6" x2="21" y2="6"></line>
                                                                                        <line x1="3" y1="18" x2="21" y2="18"></line>
                                                                                    </svg>
                                                                                </div>
                                                                                <div class="Button__Ripple-geatGG lnmdIW" style="top: 30.2188px; left: 41px;"></div>
                                                                            </div>
                                                                        </button>
                                                                    </div>
                                                                </div>
                                                                <div class="title-children-container-mobile">
                                                                    <div style="display: inline-block;">
                                                                        <button class="Button__Root-lpafWu dfGigY">
                                                                            <div class="Button__Container-eLxmbr iodBqI">
                                                                                <div class="Button__ButtonStyled-cWqOxB gHqyKq">Home</div>
                                                                                <div class="Button__Ripple-geatGG lnmdIW" style="top: 0px; left: 0px;"></div>
                                                                            </div>
                                                                        </button>
                                                                    </div>
                                                                    <div style="display: inline-block;">
                                                                        <button class="Button__Root-lpafWu dfGigY">
                                                                            <div class="Button__Container-eLxmbr iodBqI">
                                                                                <div class="Button__ButtonStyled-cWqOxB gHqyKq">About</div>
                                                                                <div class="Button__Ripple-geatGG lnmdIW" style="top: 0px; left: 0px;"></div>
                                                                            </div>
                                                                        </button>
                                                                    </div>
                                                                </div>
                                                            </div>
                                                        </div>
                                                        <div style="height: 7em; overflow: scroll;">
                                                            <h1>Line</h1>
                                                            <h1>Line</h1>
                                                            <h1>Line</h1>
                                                            <h1>Line</h1>
                                                            <h1>Line</h1></div>
                                                    </div>
                                                </div>
                                                <div></div>
                                            </div>
                                            <div class="css-ql2n7q-Playground">
                                                <button class="css-m0zonp-Playground">
                                                    <svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
                                                        <path d="M16 4h2a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H6a2 2 0 0 1-2-2V6a2 2 0 0 1 2-2h2"></path>
                                                        <rect x="8" y="2" width="8" height="4" rx="1" ry="1"></rect>
                                                    </svg>
                                                </button>
                                                <button class="css-m0zonp-Playground">
                                                    <svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
                                                        <polyline points="16 18 22 12 16 6"></polyline>
                                                        <polyline points="8 6 2 12 8 18"></polyline>
                                                    </svg>
                                                </button>
                                            </div>
                                        </div>
                                        <div class="css-nxr1xe">
                                            <div style="border-radius: 0px 0px 4px 4px; background: rgb(246, 248, 250); border-top: 0px; font-family: monospace; font-size: 18px;">
                                                <div data-testid="live-editor" style="position: relative; text-align: left; box-sizing: border-box; padding: 0px; overflow: hidden; white-space: pre; font-family: Inconsolata; color: rgb(57, 58, 52); background-color: rgb(246, 248, 250);">
                                                    <textarea class="npm__react-simple-code-editor__textarea" autocapitalize="off" autocomplete="off" autocorrect="off" spellcheck="false" data-gramm="false" style="margin: 0px; border: 0px; background: none; box-sizing: inherit; display: inherit; font-family: inherit; font-size: inherit; font-style: inherit; font-variant-ligatures: inherit; font-weight: inherit; letter-spacing: inherit; line-height: inherit; tab-size: inherit; text-indent: inherit; text-rendering: inherit; text-transform: inherit; white-space: pre-wrap; word-break: keep-all; overflow-wrap: break-word; position: absolute; top: 0px; left: 0px; height: 100%; width: 100%; resize: none; color: inherit; overflow: hidden; -webkit-font-smoothing: antialiased; -webkit-text-fill-color: transparent; padding: 10px;">&lt;div style={{ height: '10em' }}&gt; &lt;NavBar position="relative"&gt; &lt;h1&gt;Test&lt;/h1&gt; &lt;Button type="borderless"&gt;Home&lt;/Button&gt; &lt;Button type="borderless"&gt;About&lt;/Button&gt; &lt;/NavBar&gt; &lt;div style={{ height: '7em', overflow: 'scroll' }}&gt; &lt;h1&gt;Line&lt;/h1&gt; &lt;h1&gt;Line&lt;/h1&gt; &lt;h1&gt;Line&lt;/h1&gt; &lt;h1&gt;Line&lt;/h1&gt; &lt;h1&gt;Line&lt;/h1&gt; &lt;/div&gt; &lt;/div&gt;
                                                    </textarea><pre aria-hidden="true" style="margin: 0px; border: 0px; background: none; box-sizing: inherit; display: inherit; font-family: inherit; font-size: inherit; font-style: inherit; font-variant-ligatures: inherit; font-weight: inherit; letter-spacing: inherit; line-height: inherit; tab-size: inherit; text-indent: inherit; text-rendering: inherit; text-transform: inherit; white-space: pre-wrap; word-break: keep-all; overflow-wrap: break-word; position: relative; pointer-events: none; padding: 10px;"><div class="token-line" style="font-family: Inconsolata; color: rgb(57, 58, 52);"><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;</span><span class="token tag" style="color: rgb(0, 0, 159);">div</span><span class="token tag" style="color: rgb(0, 0, 159);"> </span><span class="token tag attr-name" style="color: rgb(0, 164, 219);">style</span><span class="token tag script language-javascript script-punctuation punctuation" style="color: rgb(57, 58, 52);">=</span><span class="token tag script language-javascript punctuation" style="color: rgb(57, 58, 52);">{</span><span class="token tag script language-javascript punctuation" style="color: rgb(57, 58, 52);">{</span><span class="token tag script language-javascript" style="color: rgb(0, 0, 159);"> height</span><span class="token tag script language-javascript punctuation" style="color: rgb(57, 58, 52);">:</span><span class="token tag script language-javascript" style="color: rgb(0, 0, 159);"> </span><span class="token tag script language-javascript string" style="color: rgb(227, 17, 108);">'10em'</span><span class="token tag script language-javascript" style="color: rgb(0, 0, 159);"> </span><span class="token tag script language-javascript punctuation" style="color: rgb(57, 58, 52);">}</span><span class="token tag script language-javascript punctuation" style="color: rgb(57, 58, 52);">}</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain"></span></div><div class="token-line" style="font-family: Inconsolata; color: rgb(57, 58, 52);"><span class="token plain">  </span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;</span><span class="token tag class-name" style="color: rgb(0, 0, 159);">NavBar</span><span class="token tag" style="color: rgb(0, 0, 159);"> </span><span class="token tag attr-name" style="color: rgb(0, 164, 219);">position</span><span class="token tag attr-value punctuation" style="color: rgb(57, 58, 52);">=</span><span class="token tag attr-value punctuation" style="color: rgb(57, 58, 52);">"</span><span class="token tag attr-value" style="color: rgb(227, 17, 108);">relative</span><span class="token tag attr-value punctuation" style="color: rgb(57, 58, 52);">"</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain"></span></div><div class="token-line" style="font-family: Inconsolata; color: rgb(57, 58, 52);"><span class="token plain">    </span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;</span><span class="token tag" style="color: rgb(0, 0, 159);">h1</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain">Test</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;/</span><span class="token tag" style="color: rgb(0, 0, 159);">h1</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain"></span></div><div class="token-line" style="font-family: Inconsolata; color: rgb(57, 58, 52);"><span class="token plain">    </span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;</span><span class="token tag class-name" style="color: rgb(0, 0, 159);">Button</span><span class="token tag" style="color: rgb(0, 0, 159);"> </span><span class="token tag attr-name" style="color: rgb(0, 164, 219);">type</span><span class="token tag attr-value punctuation" style="color: rgb(57, 58, 52);">=</span><span class="token tag attr-value punctuation" style="color: rgb(57, 58, 52);">"</span><span class="token tag attr-value" style="color: rgb(227, 17, 108);">borderless</span><span class="token tag attr-value punctuation" style="color: rgb(57, 58, 52);">"</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain">Home</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;/</span><span class="token tag class-name" style="color: rgb(0, 0, 159);">Button</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain"></span></div><div class="token-line" style="font-family: Inconsolata; color: rgb(57, 58, 52);"><span class="token plain">    </span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;</span><span class="token tag class-name" style="color: rgb(0, 0, 159);">Button</span><span class="token tag" style="color: rgb(0, 0, 159);"> </span><span class="token tag attr-name" style="color: rgb(0, 164, 219);">type</span><span class="token tag attr-value punctuation" style="color: rgb(57, 58, 52);">=</span><span class="token tag attr-value punctuation" style="color: rgb(57, 58, 52);">"</span><span class="token tag attr-value" style="color: rgb(227, 17, 108);">borderless</span><span class="token tag attr-value punctuation" style="color: rgb(57, 58, 52);">"</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain">About</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;/</span><span class="token tag class-name" style="color: rgb(0, 0, 159);">Button</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain"></span></div><div class="token-line" style="font-family: Inconsolata; color: rgb(57, 58, 52);"><span class="token plain">  </span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;/</span><span class="token tag class-name" style="color: rgb(0, 0, 159);">NavBar</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain"></span></div><div class="token-line" style="font-family: Inconsolata; color: rgb(57, 58, 52);"><span class="token plain">  </span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;</span><span class="token tag" style="color: rgb(0, 0, 159);">div</span><span class="token tag" style="color: rgb(0, 0, 159);"> </span><span class="token tag attr-name" style="color: rgb(0, 164, 219);">style</span><span class="token tag script language-javascript script-punctuation punctuation" style="color: rgb(57, 58, 52);">=</span><span class="token tag script language-javascript punctuation" style="color: rgb(57, 58, 52);">{</span><span class="token tag script language-javascript punctuation" style="color: rgb(57, 58, 52);">{</span><span class="token tag script language-javascript" style="color: rgb(0, 0, 159);"> height</span><span class="token tag script language-javascript punctuation" style="color: rgb(57, 58, 52);">:</span><span class="token tag script language-javascript" style="color: rgb(0, 0, 159);"> </span><span class="token tag script language-javascript string" style="color: rgb(227, 17, 108);">'7em'</span><span class="token tag script language-javascript punctuation" style="color: rgb(57, 58, 52);">,</span><span class="token tag script language-javascript" style="color: rgb(0, 0, 159);"> overflow</span><span class="token tag script language-javascript punctuation" style="color: rgb(57, 58, 52);">:</span><span class="token tag script language-javascript" style="color: rgb(0, 0, 159);"> </span><span class="token tag script language-javascript string" style="color: rgb(227, 17, 108);">'scroll'</span><span class="token tag script language-javascript" style="color: rgb(0, 0, 159);"> </span><span class="token tag script language-javascript punctuation" style="color: rgb(57, 58, 52);">}</span><span class="token tag script language-javascript punctuation" style="color: rgb(57, 58, 52);">}</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain"></span></div><div class="token-line" style="font-family: Inconsolata; color: rgb(57, 58, 52);"><span class="token plain">    </span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;</span><span class="token tag" style="color: rgb(0, 0, 159);">h1</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain">Line</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;/</span><span class="token tag" style="color: rgb(0, 0, 159);">h1</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain"></span></div><div class="token-line" style="font-family: Inconsolata; color: rgb(57, 58, 52);"><span class="token plain">    </span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;</span><span class="token tag" style="color: rgb(0, 0, 159);">h1</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain">Line</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;/</span><span class="token tag" style="color: rgb(0, 0, 159);">h1</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain"></span></div><div class="token-line" style="font-family: Inconsolata; color: rgb(57, 58, 52);"><span class="token plain">    </span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;</span><span class="token tag" style="color: rgb(0, 0, 159);">h1</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain">Line</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;/</span><span class="token tag" style="color: rgb(0, 0, 159);">h1</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain"></span></div><div class="token-line" style="font-family: Inconsolata; color: rgb(57, 58, 52);"><span class="token plain">    </span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;</span><span class="token tag" style="color: rgb(0, 0, 159);">h1</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain">Line</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;/</span><span class="token tag" style="color: rgb(0, 0, 159);">h1</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain"></span></div><div class="token-line" style="font-family: Inconsolata; color: rgb(57, 58, 52);"><span class="token plain">    </span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;</span><span class="token tag" style="color: rgb(0, 0, 159);">h1</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain">Line</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;/</span><span class="token tag" style="color: rgb(0, 0, 159);">h1</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain"></span></div><div class="token-line" style="font-family: Inconsolata; color: rgb(57, 58, 52);"><span class="token plain">  </span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;/</span><span class="token tag" style="color: rgb(0, 0, 159);">div</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span><span class="token plain"></span></div><div class="token-line" style="font-family: Inconsolata; color: rgb(57, 58, 52);"><span class="token plain"></span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&lt;/</span><span class="token tag" style="color: rgb(0, 0, 159);">div</span><span class="token tag punctuation" style="color: rgb(57, 58, 52);">&gt;</span></div></pre>
                                                    <style type="text/css">
                                                        /**
 * Reset the text fill color so that placeholder is visible
 */
                                                        
                                                        .npm__react-simple-code-editor__textarea:empty {
                                                            -webkit-text-fill-color: inherit !important;
                                                        }
                                                        /**
 * Hack to apply on some CSS on IE10 and IE11
 */
                                                        
                                                        @media all and (-ms-high-contrast: none),
                                                        (-ms-high-contrast: active) {
                                                            /**
    * IE doesn't support '-webkit-text-fill-color'
    * So we use 'color: transparent' to make the text transparent on IE
    * Unlike other browsers, it doesn't affect caret color in IE
    */
                                                            .npm__react-simple-code-editor__textarea {
                                                                color: transparent !important;
                                                            }
                                                            .npm__react-simple-code-editor__textarea::selection {
                                                                background-color: #accef7 !important;
                                                                color: transparent !important;
                                                            }
                                                        }
                                                    </style>
                                                </div>
                                            </div>
                                            <div></div>
                                        </div><span><div class="" style="position: absolute; user-select: none; width: 10px; height: 100%; top: 0px; right: -5px; cursor: col-resize;"></div></span></div>
                                    <div class="__resizable_base__" style="width: 100%; height: 100%; position: absolute; transform: scale(0, 0); left: 0px; flex: 0 1 0%;"></div>
                                </div>
                            </div>
                        </main>
                    </div>
                </div>
            </div>
        </div>
        <div id="gatsby-announcer" aria-live="assertive" aria-atomic="true" style="position: absolute; top: 0px; width: 1px; height: 1px; padding: 0px; overflow: hidden; clip: rect(0px, 0px, 0px, 0px); white-space: nowrap; border: 0px;">Navigated to NavBar Component</div>
    </div>
    <script src="/commons.js"></script>
</body>

</html>